shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/controler/CarminDataApi.java

```diff
@@ -0,0 +1,53 @@
+/**
+
+ * Shanoir NG - Import, manage and share neuroimaging data
+ * Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+ * Contact us on https://project.inria.fr/shanoir/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+package org.shanoir.ng.dataset.controler;
+
+import io.swagger.annotations.*;
+import org.shanoir.ng.dataset.model.carmin.GetPathResponse;
+import org.shanoir.ng.dataset.model.carmin.Path;
+import org.shanoir.ng.dataset.model.carmin.UploadData;
+import org.shanoir.ng.shared.exception.RestServiceException;
+import org.shanoir.ng.shared.exception.ErrorModel;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.io.IOException;
+
+/**
+ * @author Alae Es-saki
+ */
+@Api(value = "carmin-data")
+@RequestMapping("/carmin-data")
+public interface CarminDataApi {
+
+    @ApiOperation(value = "Get content or information for a given path", notes = "Download a file (or a directory) or retun information about a specific path. The response format and content depends on the mandatory action query parameter (see the parameter description). Basically, the \"content\" action downloads the raw file, and the other actions return various information in a JSON record.", tags={  })
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successful response. If the action is \"content\", the raw file (or a tarball) is returned, with the according mime type. Otherwise a json response a returned", response = GetPathResponse.class),
+            @ApiResponse(code = 200, message = "A functional or internal error occured processing the request"),
```

Comment:
Dear Alae, I hope you are doing fine. Two lines with status code 200 does not seem to be correct. Could you please adapt the change? Thank you and with kind regards, Michael

Reply:
hello Michael, I hope you are doing well, 
Can you check the new changes?
Thanks.

Reply:
Hello michael, do you have any idea about this error in github pipes ? what ive did, is only deleting one line of the apiResponse. and now the pipe can't build the application with an error in a line that doesn't exist in the file.

![image](https://user-images.githubusercontent.com/44470670/182161339-806b3fe4-deb2-4602-a92f-405c973423c6.png)

![image](https://user-images.githubusercontent.com/44470670/182161396-bb11608c-32f9-42bc-b7c6-d2736eb8f13a.png)

Reply:
when i run the build on my machine it works fine 

![image](https://user-images.githubusercontent.com/44470670/182163763-3ed22673-4aba-4598-96e8-71de2bc8a36f.png)

Reply:
@michaelkain , i rolledback to the last working pipe commit, and the error is still showing.
what do you think ?

Reply:
hello @michaelkain , i've found that in the dev branch, there is a new calling for a method moved to the utility file in this PR, so the compiler doesn't recognise it
I pulled the dev and it's fixed.